utils/compute_pool_address.go

```diff
@@ -10,6 +10,10 @@ import (
 	"github.com/ethereum/go-ethereum/crypto"
 )
 
+var (
+	UniswapV3FactoryAddress = common.HexToAddress("0x1f98431c8ad98523631ae4a59f267346ea31f984")
```

Comment:
on BSC, the factory is a different address, so I think this is not a good solution